docs/examples/Tutorials/Tutorial4/DeviceGroupQuery.cs

```diff
@@ -26,6 +27,8 @@ public DeviceGroupQuery(Dictionary<IActorRef, string> actorToDeviceId, long requ
                 Timeout = timeout;
 
                 queryTimeoutTimer = Context.System.Scheduler.ScheduleTellOnceCancelable(timeout, Self, CollectionTimeout.Instance, Self);
+                
+                Become(WaitingForReplies(new Dictionary<string, ITemperatureReading>(), new HashSet<IActorRef>(ActorToDeviceId.Keys)));
```

Comment:
You can just change this to `WaitingForReplies` without the `Become` syntax here

Reply:
The output of `WaitingForReplies` is a factory method that will be used as input of `Context.Become` method. The author use it in the same way at row 116.